pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -205,7 +218,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 
 				// backwards compatibility
-				authorizationapi.NewRule(read...).Groups(buildGroup).Resources("buildlogs").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(buildGroup, legacyBuildGroup).Resources("buildlogs").RuleOrDie(),
```

Comment:
Newer clients should know not to rely to legacy behavior - granted they may need legacy behavior to interact with older servers...